<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>streamLineFilterPropForm</class>
 <widget class="QWidget" name="streamLineFilterPropForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Streamline Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>vectors</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="vector_comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Interpolator Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="interpolatorType_comboBox">
       <item>
        <property name="text">
         <string>Interpolator with Point Locator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interpolator with Cell Locator</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="surfaceLines_checkBox">
     <property name="text">
      <string>Surface Streamlines</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Integration Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Integration Direction</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Inte_Direction">
        <item>
         <property name="text">
          <string>FORWARD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BACKWARD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BOTH</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Integrator Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="inte_Type">
        <item>
         <property name="text">
          <string>Runge-Kutta 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Runge-Kutta 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Runge-Kutta 4-5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Integration Step Unit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="inte_StepUnit">
        <item>
         <property name="text">
          <string>Length</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cell Length</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Initial Step Length</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="inte_InitStepLen">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Minimum Step Length</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="inte_MiniStepLen">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Maximum Step Length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="inte_MaxiStepLen">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Streamline Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Maximum Steps</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="stream_MaxiSteps">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Maximum Streamline Length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="stream_MaxiLen_horizontalSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stream_MaxiLen_lineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Terminal Speed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="stream_TerminalSpeed">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Maximum Error</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="stream_MaxiError">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="vorticity_checkBox">
     <property name="text">
      <string>Compute Vorticity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Seeds</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Seed Types</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="seeds_Type">
        <item>
         <property name="text">
          <string>Point Source</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High Resolution Line Source</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Point</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="seedP1_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="seedP2_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="seedP3_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Number of Points</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="seedPoints_spinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="seedRadius_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reflection_checkBox">
     <property name="text">
      <string>Reflection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="reflection_groupBox">
     <property name="title">
      <string>reflection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>reflection:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="reflection_comboBox">
        <item>
         <property name="text">
          <string>X_MIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y_MIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z_MIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X_MAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y_MAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z_MAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
